elpis/wrappers/objects/dataset.py

```diff
@@ -41,8 +44,8 @@ def __init__(self, basepath: Path):
         self.word_list_txt: Path = self.basepath.joinpath('word_list.txt')
         # \/ user uploaded addional words
```

Comment:
Does this: "\\/" signify something important @mcqueen256 ?

Reply:
or is it some kind of emoji?